<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EP_Main</class>
 <widget class="QMainWindow" name="EP_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expense Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxLogIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Log In</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelLogInUsername">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LogInUsername"/>
     </item>
     <item>
      <widget class="QLabel" name="labelLogInPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LogInPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLogIn">
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonConnectDB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT DB 
BEFORE ENTERING ANYTHING</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateNewAccount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create a new account</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>202</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
